README.md

```diff
@@ -59,6 +59,33 @@ added to the resources/enfsi folder.
 
 To use the streamlit app, write in terminal: `streamlit run run_data_exploration.py`
 
+### Finetuning the model
+##### Setup DGX / Docker for training and evaluating the model
+
+Building the container
+```bash
+docker build . -f Dockerfile -t <image_name>
+```
+
+Pushing the container to the DGX
+```bash
+docker tag <image_name> dgxstation01.holmes.nl:5000/<image_name>
```

Comment:
ik zou voorstellen om dit soort adressen in lokale yaml files te zetten.

Reply:
ik vind het een beetje gek om dingen voor de readme in een yaml te zetten, dus ik heb er <dgx_address> van gemaakt, en die staat in principe gewoon op onze confluence

Reply:
top